lib/widgets/atoms/user_search_bar.dart

```diff
@@ -11,6 +11,17 @@ class UserSearchBar extends StatelessWidget {
     return SearchBar(
       leading: const Icon(Icons.search),
       hintText: hintText,
+      hintStyle: MaterialStateProperty.all(TextStyles.searchText(context)),
+      backgroundColor: MaterialStateProperty.all(
+          Theme.of(context).colorScheme.primaryContainer),
+      padding: MaterialStateProperty.all(
+        const EdgeInsets.fromLTRB(
+            Insets.widgetLargeInset,
+            Insets.widgetSmallInset,
+            Insets.widgetSmallInset,
+            Insets.widgetSmallInset),
+      ),
+      // shape: MaterialStateProperty.all({BorderSide side: BorderSide.none,}),
```

Comment:
I don't see this Widget being used anywhere else in the PR. Why is this change included?

Reply:
Ah, great catch! This looks like I may have merged the widget from the other branch I'm working on for the search bar on the explore page. I will remove this widget from this PR. Thanks!

Reply:
I have removed the file from the PR.